cluster/src/dunit/scala/io/snappydata/cluster/ClusterManagerTestBase.scala

```diff
@@ -121,7 +121,8 @@ abstract class ClusterManagerTestBase(s: String)
       }
     }
 
-    Array(vm0, vm1, vm2).foreach(_.invoke(startNode))
+    vm0.invoke(startNode)
+    Array(vm1, vm2).map(_.invokeAsync(startNode)).foreach(_.getResult)
```

Comment:
Didn't note the total precheckin saving. But saw good improvement when running a single dunit.